src/foam/u2/Element.js

```diff
@@ -523,7 +523,7 @@ foam.CLASS({
   listeners: [
     {
       name: 'paint',
-      isMerged: 100,
+      isFramed: true,
```

Comment:
This isn't strictly necessary, but it fixes an issue where the table wasn't re-rendering until the end of the scroll.